<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>301</width>
      <height>217</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="add_layout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Добавить запись:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Название ведомости:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_name_map_add"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Имя студента:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_student"/>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Его результат:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_value"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_add">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>290</y>
      <width>301</width>
      <height>107</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="best_3_layout">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Отобразить 3 лучших студента:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Название ведомости:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_name_map_best"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_result_best">
       <property name="text">
        <string>Отобразить результат</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>301</width>
      <height>162</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="delete_layout">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Удалить:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Название ведомости:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_name_map_del"/>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Имя студента:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_student_2"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delete">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>921</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>921</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Название ведомости:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_name_map"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_load">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Загрузить</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_save">
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_result">
         <property name="text">
          <string>Отобразить ведомость</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>921</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Консоль</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Все ведомости</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>921</width>
      <height>194</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QTextBrowser" name="textBrowser_console"/>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser_all_map"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
